lib/rev-api/models/order.rb

```diff
@@ -55,51 +49,43 @@ def initialize(fields)
     end
   end
 
-  # Additional information specific to translation orders,
-  # such as word count, languages
-  class TranslationInfo < ApiSerializable
-    attr_reader :total_word_count, :source_language_code,
-      :destination_language_code
-  end
-
   # Additional information specific to transcription orders,
   # such as total length in minutes, verbatim and timestamps flags
   class TranscriptionInfo < ApiSerializable
     attr_reader :total_length_seconds, :verbatim, :timestamps
-    
+
     # @deprecated use :total_length_seconds instead
     attr_reader :total_length
   end
-  
+
   # Additional information specific to caption orders
   class CaptionInfo < ApiSerializable
     attr_reader :total_length_seconds
-    
+
     # @deprecated use :total_length_seconds instead
     attr_reader :total_length
   end
 
   # Represents order attachment - logical document associated with order
   class Attachment < ApiSerializable
-    attr_reader :kind, :name, :id, :audio_length_seconds, :word_count, :links, :video_length_seconds
-    
+    attr_reader :kind, :name, :id, :audio_length_seconds, :links, :video_length_seconds
+
     KINDS = {
       :transcript => 'transcript',
-      :translation => 'translation',
       :caption => 'caption',
       :media => 'media'
     }
 
     # List of supported mime-types used to request attachment's content
     # within 'Accept' header
     REPRESENTATIONS = {
-      # Supported by :transcript and :translation
+      # Supported by :transcript
       :docx => 'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
       :doc => 'application/msword',
       :pdf => 'application/pdf',
       :txt => 'text/plain',
       :youtube => 'text/plain; format=youtube-transcript',
-      
+
       # Supported by :caption
       :srt => 'application/x-subrip',
```

Comment:
unrelated - this list is woefully incomplete.  can you add in all the supported types?